src/hiero_sdk_python/hbar.py

```
@@ -107,6 +107,54 @@ def from_tinybars(cls, tinybars: int) -> "Hbar":
         if not isinstance(tinybars, int):
             raise TypeError("tinybars must be an int.")
         return cls(tinybars, unit=HbarUnit.TINYBAR)
+    
+    @classmethod
+    def from_microbars(cls, amount: Union[int, float, Decimal]) -> "Hbar":
```

Ruff (0.14.10)

Use `X | Y` for type annotations

Convert to `X | Y`

(UP007)